Library/Homebrew/command.rb

```diff
@@ -72,13 +72,51 @@ def self.desc(desc)
       end
     end
 
+    def self.get_invalid_options(argv_options_only)
+      argv_options_only = argv_options_only.uniq
+      valid_options_with_values =
+        @valid_options
+        .select { |option_hash| !option_hash[:value].nil? }
+        .map { |option_hash| option_hash[:option] }
+      all_valid_options =
+        @valid_options
+        .map { |option_hash| option_hash[:option] }
+
+      argv_invalid_options = []
+      argv_options_without_value = {}
+      argv_options_only.each do |argv_option|
+        if argv_option.include?("=")
```

Comment:
Using `String#split` would make this a lot easier to read:

```ruby
x = "--foo=bar=baz"
x.split("=", 2) # => "--foo", "baz=baz"
```

The `2` argument above prevents us from splitting beyond the first `=`, just in case we ever need that in a value for some reason.